pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -0,0 +1,38 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore": "5.0.1-rc3-24126-00",
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc3-24126-00",
+    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24126-00",
```

Comment:
However, the feed will soon have the CoreCLR package being published from open, which requires reverse forwarders to be inplace. Since that will not work on Win10, by simply moving this definition out, we are breaking that support.

Prior to the feed changes, the package came from closed build, which would have just worked due to api-set linkage. Should there be a plan for defining a RID for Win10 as we discussed in the past?

Also, given that CoreCLR gets installed via the MSI for UWP, what does it mean to pulling in this package?